web/modules/custom/usagov_ssg_postprocessing/src/EventSubscriber/PublishedPagesSubscriber.php

```diff
@@ -118,7 +119,7 @@ public function modifyHtml(ModifyHtmlEvent $event) {
         $nodeIDElement = array_search("Page ID", $csv[0]);
         $languageElement = array_search("Taxonomy Level 1", $csv[0]);
         foreach ($csv as $key => $line) {
-          if ($line[$nodeIDElement] == $decoded["nodeID"]) {
+          if (!empty($line[$nodeIDElement]) && $line[$nodeIDElement] == $decoded["nodeID"]) {
```

Comment:
We need to repeat this test for taxonomyIDs, or otherwise expand this clause. (And maybe it would be good to change the name of that variable from $nodeIDElement to $pageIDElement or something.) 

What's missing is a test for the case where there's a taxonomyID, say "60," and there's already an entry in the PPR for it -- that would have "t_60" in the Page ID position. 

I was able to reproduce the problem by first, having a wizard taxonomy term on my local *and* setting its path alias (because otherwise I think it gets omitted, due to the /taxonomy/term path). Run tome and get the published_pages.csv file. The edit and re-save the wizard term, run tome again (happily, it goes pretty fast when you've just updated one term like that). Now I have two entries for that term in my published_pages.csv.

Reply:
oh! I see this now - in fact, every tome-run was adding a new taxonomy-term to a new line..
I have updated, and this issue is now fixed with the latest commit.

Reply:
Oh, hmm, I may not be as familiar with our workflow here. I previously clicked the "Resolve conversation button", but now I am assuming that perhaps that is something that Amy does and I instead am supposed to request a re-review? Let me know if I am doing this right.

Reply:
You did the right thing, and I just got bogged down and didn't have time to do the re-review until today.